contracts/test/utils/TestGatewayCaller.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+contract TestGatewayCaller {
+    function delegatecall(bytes memory) external pure returns (bool, bytes memory) {
+        // solhint-disable-next-line reason-string
+        revert();
```

Comment:
address this lint. you can define an error code

Reply:
defined an error code `delegatecallFailed`. updated.